src/mono/wasm/build/WasmApp.Native.targets

```diff
@@ -674,12 +674,25 @@
       CacheFilePath="$(_AOTCompilerCacheFile)"
       LLVMDebug="dwarfdebug"
       LLVMPath="$(EmscriptenUpstreamBinPath)"
+      CollectCompiledMethods="$(WASMStripIL)"
+      CompiledMethodsOutputDirectory="$(_WasmIntermediateOutputPath)tokens"
       IntermediateOutputPath="$(_WasmIntermediateOutputPath)"
       AotProfilePath="@(AotProfilePath)">
 
       <Output TaskParameter="CompiledAssemblies" ItemName="_WasmAssembliesInternal" />
       <Output TaskParameter="FileWrites" ItemName="FileWrites" />
     </MonoAOTCompiler>
+    <ILStrip
+        Condition=" '$(WASMStripIL)' == 'true' "
+        TrimIndividualMethods="true"
```

Comment:
Make the value for this a property too.

Reply:
I don't think this is necessary. `MonoAOTCompiler` also has parameters set to either `true` or `false` on the spot. I prefer to do it this way to provide more clarity for future maintenance.